frontend/tests/unit/stores/settings.spec.ts

```diff
@@ -11,17 +11,26 @@ describe('settings store', () => {
     setActivePinia(pinia);
   });
   describe('Settings initial value', () => {
-    it('initialize settings store with undefined values', () => {
+    it('initialize settings store with default values', () => {
       const settings = useSettings();
       expect(settings.$state.connectedWallet).toBeUndefined();
+      expect(settings.$state.disclaimerChecked).toBe(false);
```

Comment:
When writing tests i prefer to find solutions where i only do a single `expect` check (unless i am checking against different values for the same target). 
In this case, i would separate the `disclaimerChecked` and `connectedWallet` checks into their own test.
With the current approach it will make it harder to debug when something breaks in this test. Of course at this moment it will be pretty simple to debug since we have only 2 cases we are testing but if we add more store variables it can get messier here.

Reply:
Cool makes sense to me :+1: